SEP-0012.md

```diff
@@ -298,14 +298,9 @@ class NDCubeABC(astropy.nddata.NDDataBase):
         """
         Crop given world coordinate objects describing the lower and upper corners of a region.
```

Comment:
```suggestion
        Crop the cube to a region that includes given world coordinates defined by high-level objects.
```

This frees us from explicitly committing to the inputs representing corners. All this commits us to is that the cropped region will include the input coordinates.